### ISSUE TITLE
Make sure "Prioritized Background Activity" is enabled under Android

### ISSUE THREAD

**Issue by daimpi** [corona-warn-app/cwa-app-android#925]
## Avoid duplicates
checked

## Current Implementation
It is great that the App has now (v. 1.1.1) a setting for users to exempt it from the overly aggressive killing of apps that many device manufacturers have implemented.
One problem I see with the current implementation is that “prioritized background activity” is by default set to “off” in the App. Users therefore have to first go to the settings menu and manually turn this setting on. There is also no indication on the main screen of the app that anything could be amiss when this setting is off, which makes it even less likely that users would realize the problem. I spot checked a few phones from family/friends and found that on almost all devices the number of checks performed in the Google ENF settings was much lower than it should have been (sometimes only having performed a check today when I opened the app manually). And looking at https://dontkillmyapp.com/ I’m afraid that this is not an outlier experience. Also: only one of them had manually activated “prioritized background activity”, everyone else didn’t even know about this and I had to manually activate it for them.

The Phones checked were: 
- 2x Samsung Galaxy S8 (Android 9) (one of them had manually activated the setting just earlier today)
- Lg G5 (Android 8.1) (This was the only phone where the checks were performed every day)
- Xiaomi Redmi 5 Plus (Android 8.1)
- Xiaomi Redmi Note 7 (Android 9)
- Samsung Galaxy S10e (Android 10)

All were running CWA v. 1.1.1

## Suggested Enhancement
Activate "Prioritized Background Activity" during the onboarding process (and after updating) and add a status indication for this setting to the main screen with a warning that something is wrong when it is turned off.

## Expected Benefits
One of the expected benefits from the app is the speed with which contacts can be notified, which becomes moot if the app is killed and therefore cannot synchronize with the server in the background to find out whether anyone it had contact with has actually tested positive. This is particularly bad b/c many user will not regularly check the app and might never learn about their risk contacts. Also: many people seem to be unaware of the problem and are unlikely to be checking the settings of the app where they have to activate the toggle. 
Making sure that "Prioritized Background Activity" is activated for all users therefore seems crucial to ensure proper functionality of the App.

---
Internal Tracking ID: EXPOSUREAPP-1927

**Comment by ubuntudroid** [comment]
There also is a library which helps routing the user to related settings screens on many devices: https://github.com/judemanutd/AutoStarter

At the core it's just just a collection of setting intents.

**Comment by daimpi** [comment]
Today I noticed something strange on my phone: even though I enabled Prioritized Background Activity (PBA) early on, it was working fine for months and I never disabled it, I found that it was disabled today when I checked my CWA settings by chance. When re-enabling PBA in the CWA settings I encountered https://github.com/corona-warn-app/cwa-app-android/issues/906.
My device:
- Samsung Galaxy S8 (Android 9)
- CWA 1.3.1


Something similar [happened](https://github.com/corona-warn-app/cwa-app-android/issues/933#issuecomment-702894697) on a device in my family (Xiaomi Redmi 5 Plus, Android 8.1, CWA 1.3.1) as well.
